config/rails.yml

```diff
@@ -77,9 +77,12 @@ Layout/EmptyLinesAroundMethodBody:
 Layout/EmptyLinesAroundModuleBody:
   Enabled: true
 
-Layout/FirstParameterIndentation:
+Layout/IndentFirstArgument:
   Enabled: true
-
+  
+Layout/IndentFirstParameter:
+  Enabled: true
```

Comment:
This cop is not added in rails/rails.
https://github.com/rails/rails/pull/36426

Reply:
Gone!